src/back/TicketManager/src/Core/TicketManager.Core.Services/Extensions/QueryableExtensions.cs

```diff
@@ -27,4 +29,54 @@ public static class QueryableExtensions
             TotalCount = count,
         };
     }
+
+    public static IQueryable<T> ToOrderedQueryable<T, TKey, TSort>(this IQueryable<T> query,
```

Comment:
Maybe rename to just `OrderBy`?
Also return `IOrderedQuerable` in case someone wants to call `ThenBy` on it.

Reply:
We could rename it, but `OrderBy` suggests increasing order as per usual LINQ

Reply:
So maybe `SortBy`? It means the same as `OrderBy`, but it's not used in linq.